discuss/overview/team_communication.rst

```diff
@@ -0,0 +1,117 @@
+======================================
+Efficiently Communicate Using Channels
+======================================
+
+You can use channels to organize discussions between individual teams, departments, projects, or any
+other group that requires regular communication. This way, you keep everyone in the loop updated
+with the latest developments.
```

Comment:
I would point out that Discuss is an excellent internal communication tool.

Reply:
I believe, and this is the intention, that this is exactly what the paragraph says, explaining what it is a great internal communication tool (: